packages/contracts/src/interest/IMultiTokenVault.sol

```diff
@@ -28,29 +85,35 @@ interface IMultiTokenVault is IERC1155 {
     function asset() external view returns (address);
 
     /**
-     * @dev Returns the total amount of the underlying asset that is held by vault.
+     * @dev Returns the total amount of the underlying asset that is managed by vault.
      *
-     * @return totalAssets The total amount of the underlying asset.
+     * @return totalAssets The total amount of the underlying asset that is managed by vault.
      */
-    function totalAssets() external view returns (uint256 totalAssets);
-
-    function sharesAtPeriod(address account, uint256 depositPeriod) external view returns (uint256 shares);
+    function totalAssets() external view returns (uint256);
```

Comment:
i don't think either of our implementations were consistent with the IERC4626 definition (see Natspec below).  to calculate correctly we would need to calculate returns across all periods.  this seems to be a pretty "heavy" call.  maybe we can simplify and provide totalAssets(depositPeriod) ?  

    /**
     * @dev Returns the total amount of the underlying asset that is “managed” by Vault.
     *
     * - SHOULD include any compounding that occurs from yield.
     * - MUST be inclusive of any fees that are charged against assets in the Vault.
     * - MUST NOT revert.
     */
    function totalAssets() external view returns (uint256 totalManagedAssets);